lib/api/1.1/northbound/router.js

```diff
@@ -0,0 +1,57 @@
+// Copyright 2015, EMC, Inc.
+
+'use strict';
+
+var di = require('di'),
+    express = require('express');
+
+module.exports = northApiFactory;
+
+di.annotate(northApiFactory, new di.Provide('northbound-api-router'));
+di.annotate(northApiFactory, new di.Inject(
+        di.Injector
+    )
+);
+
+function northApiFactory (injector) {
+    var router = express.Router();
+    router.endpoint = {};
+
+    router.mount = function() {
```

Comment:
Yes, depends on the effort. I guess add an underscore to the name should also be safe.